test/features/deploy_quarkus_service.feature

```diff
@@ -4,13 +4,44 @@ Feature: Deploy quarkus service
   Background:
     Given Namespace is created
 
+  @test
```

Comment:
I guess this is just a remainder of WIP?

Reply:
oops ! corrected :)